skops/cli/tests/test_entrypoint.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+import sys
+from unittest import mock
+
+import pytest
+
+from skops.cli.entrypoint import main_entrypoint
+
+
+class TestEntrypoint:
+    """Integration tests that check that entrypoint calls pass through correctly.
+
+    Full coverage of individual entrypoint calls should be done in their own classes.
+    """
+
+    @pytest.fixture(autouse=True)
+    def clear_argv(self):
+        # Required to clear argv in case Pytest is called on this specific function.
+        # Otherwise, clogs parser.parse_known_args() in argparse
+        sys.argv = [""]
+
+    @mock.patch("skops.cli._convert._convert_file")
+    def test_convert_works_as_expected(self, mocked_convert: mock.MagicMock):
```

Comment:
As this is meant to be more of an "end-to-end" test, how about testing it without mocking? Otherwise, I don't think it adds a lot of value, given the other tests, and could be removed.

Reply:
Sure, happy to do it that way

When I've written CLIs before in other projects, the standard pattern was to write unit tests for the functionality (here: `_convert_file`) and also unit tests for the interface itself (here: `entrypoint`, and checking it interfaces correctly with `_convert_file`)

But for a simple entrypoint, the happy path test is pretty simple, so happy to rework this one as an "end-to-end" test for the full functionality.

Reply:
I see the idea here, but logically, what bug could be prevented here that is not already covered by the previous tests? In contrast, when not mocking, there could be a case that this tests that everything is glued together correctly. But that's not very likely to be broken, which is why I would also be fine with removing this test, given the other ones above.

Reply:
I think it would be good to have a full integration test of some sort, that might be a good first contributor issue we can raise after this PR.